src/pyclaw/sharpclaw/solver.py

```diff
@@ -397,16 +391,25 @@ def step(self,solution):
             return False
 
 
+    def ssp22(self,state,dq_dt):
```

Comment:
It seems confusing to me that this function (and ssp104) do *both* of the following:

- Modify state.q in place
- Return a q array

It seems like they should do one or the other, but not both.